doc/man/scons.xml

```diff
@@ -3974,6 +3974,11 @@ If omitted, the default stanza will be
 (with <parameter>function_name</parameter> appropriately substituted):
 </para>
 
+    <para>
+      The optional <parameter>funcargs</parameter> can be defined to specify an argument list for the generated
```

Comment:
This addition should go after the existing example.  Also notice that further down it basically says "don't do that" about what this change is enabling (that is, including a standard header).  We'll want to think about what that should say now.  Also include a version notics.

Reply:
Updated text. Please take another pass at this?

Reply:
Not seeing anything different. Is github interface messing up again?